pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -43,6 +43,55 @@ func TestParseLocation(t *testing.T) {
 	assert.Equal(t, "example.com:5000/with/path", location)
 }
 
+func TestParseSubDomainPrefix(t *testing.T) {
+	var err error
+	var subdomainprefix string
+
+	// invalid subdomainprefix
+	_, err = parseSubDomainPrefix("https://example.com")
```

Comment:
Please use table-driven tests as in `TestRefMatchesSubDomainPrefix`. They are easier to brain parse and to maintain.

Reply:
ack, I guess this comment would also apply to `TestParseLocation`, yes? I guess I could address that in a separate commit.

Reply:
addressed in upcoming change.